pkgs/cli_util/lib/src/base_directories.dart

```diff
@@ -0,0 +1,274 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io' show Platform;
+
+import 'package:path/path.dart' as path;
+
+import '../cli_util.dart';
+
+/// The standard system paths for a Dart [tool].
```

Comment:
... for a Dart tool.

The `[tool]` links to "The name of the Dart tool". The name is not the thing.